scripts/compute_average_distance.py

```
@@ -0,0 +1,572 @@
+        factor = FALLBACK_FACTOR.get(line_type, 1.15)
+        res.average = res.straight_avg * factor
+    res.method = "fallback"
+    return res
+
+
+# ---------------------------------------------------------------------------
+# CSV 書き換え
+# ---------------------------------------------------------------------------
+def format_distance(value: float) -> str:
+    """距離値を既存 CSV と同程度の桁数の文字列に整形する。"""
+    # 既存CSVは小数を含む（例: 31785.47337）。同程度の桁で出力する。
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)